packages/twenty-apps/__tests__/validate-apps.test.ts

```diff
@@ -0,0 +1,137 @@
+import Ajv, { type ValidateFunction } from 'ajv';
```

Comment:
Let's remove this file for now and create more robust tests once everything is set in stone?